packages/geo/src/bounds.ts

```diff
@@ -160,9 +163,9 @@ export class Bounds implements BoundingBox {
      * Round dimensions to integers keeping the error a low as possible
      */
     public round(): Bounds {
```

Comment:
Do we want to do this for all Rounds?

do we want to `round(bias:number = 0.2)` ? or `round(bias?:number)` and default to 0 then explicitly put the bias in?

Reply:
or even `roundWithBias()`